helpers/decimals.ts

```diff
@@ -0,0 +1,10 @@
+export const roundToDecimalPlaces = (number, decimalPlaces) => {

```

Comment:
consider adding come comments with explanation, for example using this scheme https://tsdoc.org

Reply:
Sure, a good idea, added.